pkg/synthetictests/platformidentification/types.go

```diff
@@ -6,25 +6,31 @@ import (
 
 	configv1 "github.com/openshift/api/config/v1"
 	configclient "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	exutil "github.com/openshift/origin/test/extended/util"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/rest"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
 )
 
+var oc = exutil.NewCLIWithoutNamespace("default")
```

Comment:
no.  This is a library type.

Reply:
It is used [here](https://github.com/openshift/origin/pull/26849/files#diff-a395e190c5afd27cbc9248b8604043769a47e9ae58909aa8a69b95c8ab2a57b0R125)